examples/custom-theme/app/Providers.tsx

```diff
@@ -7,7 +7,13 @@ export default function Providers({ children }: { children: React.ReactNode }) {
   const queryClient = new QueryClient()
   return (
     <QueryClientProvider client={queryClient}>
-      <StoryKitProvider chain={STORYKIT_SUPPORTED_CHAIN.STORY_TESTNET} theme="my-theme">
+      <StoryKitProvider 
```

Comment:
I'm not sure if `StoryKitProvider` is appropriate for API environment
It seems like those env vars are only needed for IP-related API calls. How about... move to `IpProvider`?